charts/kyverno/templates/crds.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.4.0
     config.kubernetes.io/index: '1'
+    internal.config.kubernetes.io/index: '1'
```

Comment:
What are these index annotations? Can we remove them?

Reply:
The make kustomize-crd command seems to have added them. Looks like it was previously `config.kubernetes.io/index: '1'` - see line above.